draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -80,7 +66,7 @@ code and issues list for this draft can be found at
 
 HTTP clients need to know that the content they receive on a connection comes
 from the origin that they intended to retrieve in from. The traditional form of
-server authentication in HTTP has been in the form of X.509 certificates
+server authentication in HTTP has been in the form of the single X.509 certificate
```

Comment:
s/the/a/ ?

Reply:
Perhaps; "the" when there's only one, and the point of the statement is that there's only one.  But maybe I'm over-emphasizing that?

Reply:
Yeah, "the single" is redundant in that sense.